diff --git a/node_modules/sip.js/lib/core/user-agent-core/user-agent-core.js b/node_modules/sip.js/lib/core/user-agent-core/user-agent-core.js
index 18b4257..1f143b1 100644
--- a/node_modules/sip.js/lib/core/user-agent-core/user-agent-core.js
+++ b/node_modules/sip.js/lib/core/user-agent-core/user-agent-core.js
@@ -272,6 +272,8 @@ export class UserAgentCore {
                     // We are not currently supporting this case.
                     // NOTE: Not backwards compatible with RFC 2543 (no support for strict-routing).
                     this.logger.warn(`Discarding out of dialog ACK after 2xx response sent on transaction ${transactionId}.`);
+                    // this.receiveInsideDialogRequest(message);
+                    this.receiveRequest(message);
                     return;
                 }
             }
diff --git a/node_modules/sip.js/lib/platform/web/session-description-handler/peer-connection-configuration-default.js b/node_modules/sip.js/lib/platform/web/session-description-handler/peer-connection-configuration-default.js
index 2ab1320..2a6da65 100644
--- a/node_modules/sip.js/lib/platform/web/session-description-handler/peer-connection-configuration-default.js
+++ b/node_modules/sip.js/lib/platform/web/session-description-handler/peer-connection-configuration-default.js
@@ -7,10 +7,10 @@ export function defaultPeerConnectionConfiguration() {
         bundlePolicy: "balanced",
         certificates: undefined,
         iceCandidatePoolSize: 0,
-        iceServers: [{ urls: "stun:stun.l.google.com:19302" }],
+        iceServers: [{ urls: "stun:stun.l.google.com:19302" }, { urls: "stun:stun.voiparound.com:3478" }],
         iceTransportPolicy: "all",
         peerIdentity: undefined,
-        rtcpMuxPolicy: "require"
+        rtcpMuxPolicy: "negotiate"
     };
     return configuration;
 }
diff --git a/node_modules/sip.js/lib/platform/web/session-description-handler/session-description-handler-factory-default.js b/node_modules/sip.js/lib/platform/web/session-description-handler/session-description-handler-factory-default.js
index cf2bd48..a628957 100644
--- a/node_modules/sip.js/lib/platform/web/session-description-handler/session-description-handler-factory-default.js
+++ b/node_modules/sip.js/lib/platform/web/session-description-handler/session-description-handler-factory-default.js
@@ -16,7 +16,7 @@ export function defaultSessionDescriptionHandlerFactory(mediaStreamFactory) {
             mediaStreamFactory = defaultMediaStreamFactory();
         }
         // make sure we allow `0` to be passed in so timeout can be disabled
-        const iceGatheringTimeout = (options === null || options === void 0 ? void 0 : options.iceGatheringTimeout) !== undefined ? options === null || options === void 0 ? void 0 : options.iceGatheringTimeout : 5000;
+        const iceGatheringTimeout = (options === null || options === void 0 ? void 0 : options.iceGatheringTimeout) !== undefined ? options === null || options === void 0 ? void 0 : options.iceGatheringTimeout : 1000;
         // merge passed factory options into default session description configuration
         const sessionDescriptionHandlerConfiguration = {
             iceGatheringTimeout,
